### PR TITLE
Document date command

### PR DESCRIPTION
**Pull request by marcelocg**
Here is my first contribution to commands documentation.

Please let me know what you think of the style, formatting, depth of the content, etc

### CODE REVIEW

**Comment by jonathandturner** [comment]
Looks good. I just had one small wording change.

File: docs/commands/date.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# date
+
+Use `date` to get the current date and time. Defaults to local timezone but you can get it in UTC too.
+
+## Flags
+
+    --utc
+      Returns da current date and time in UTC
```
Comment (coolshaurya) [inline docs/commands/date.md]: Use the instead of da

File: docs/commands/date.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    --local
+      Returns da current date and time in your local timezone
```
Comment (coolshaurya) [inline docs/commands/date.md]: Use 'the' instead of 'da'

File: docs/commands/date.md (hunk 1 above)
Comment (marcelocg) [reply docs/commands/date.md]: Fixed! Thanks!

File: docs/commands/date.md (hunk 2 above)
Comment (marcelocg) [reply docs/commands/date.md]: Fixed! Thanks!

**Comment by jonathandturner** [comment]
Looks good!

**Pull request merged**